tests/acceptance/test_multi_threading.py

```diff
@@ -200,6 +200,7 @@ def test___shared_interpreter___run_multiple_acquisitions_with_events___callback
         assert all(status == 0 for status in done_statuses)
 
 
```

Comment:
As per the fix made in https://github.com/ni/grpc-device/pull/940, this test requires the latest gRPC device server.